orchestra/static/orchestra/timing/timecard/date-picker/date-picker.html

```diff
@@ -1,3 +1,3 @@
 <span class="date-picker">
-  <i class="fa fa-calendar-o"></i>{{ date.format('MMMM D') }}
+  <i class="fa fa-calendar-o"></i>{{ date.format(' ddd MMMM D') }}
```

Comment:
prepended space on purpose?

Reply:
looking at https://momentjs.com/, should this be `dddd`?

Reply:
ddd shows the 3-letter day of week (e.g. Tue, Thr, ...). It is more compact. The space is added for aesthetic.